barman/infofile.py

```diff
@@ -955,8 +955,11 @@ def __init__(self, server, backup_dir, info_file=None, **kwargs):
         # the backup_id is then the directory name where the backup resides
         backup_id = os.path.basename(os.path.normpath(backup_dir))
 
-        super(SynthethicBackupInfo, self).__init__(
-            server=server, info_file=info_file, backup_id=backup_id, **kwargs
+        super(SyntheticBackupInfo, self).__init__(
```

Comment:
I think i did not understand this part. 
we are going to have a command like `barman recover [OPTIONS] SERVER_NAME BACKUP_ID DESTINATION_DIRECTORY`. Why use backup_dir here? The DESTINATION_DIRECTORY is something like path/to/backup and backup_id is already passed here. Can't we just use those directly?